### ISSUE TITLE
[Bug] Crash when placing pipes near other players.

### ISSUE THREAD

**Issue by DoughnutDev** [CrafterKina/Pipes#26]
Description
---
When placing a pipe in multiplayer with another player present they will crash out to desktop. Here is there crashlog. https://hastebin.com/dibagiyiva.pas

**Comment by CrafterKina** [comment]
> jp.crafterkina.pipes{1.1.0.8} [Pipes] (Pipes-1.1.0.8.jar) 

You are using an old version.
Latest is 1.1.0.10, so please use it.

**Comment by Kriptikz** [comment]
Sorry, but this also just happened to me on my server and feel this is related so posting here instead of a new issue. 
[http://pastebin.com/awCsZ6xj](http://pastebin.com/awCsZ6xj)

I don't know exactly what caused this. I was placing diamond pipes around my base to extract from some actually additions farmers into rftools modular storages and after some time I noticed one of the pipes looked like an iron one, which I didn't even make any of, so I went over to it to check it out and break it and see what was going on. The moment I got close enough, I think for hwyla or waila harvestability to get data from the pipe, I crashed. This didn't crash the server.
I'm sorry I should have done more testing but I didn't want this to cause any problems for anyone else on the server so I immediately deleted this world and restored from a backup.
